vars/computeGitData.groovy

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/groovy
 
 def Map call() {
-  def owner = env.GITHUB_REPOSITORY_OWNER
+  def owner = env.GITHUB_REPOSITORY_OWNER ?: env.GITHUB_REPOSITORY_OWNER_NAME
```

Comment:
Why not keeping only env.GITHUB_REPOSITORY_OWNER?

Reply:
because we have 2 sources with different names travis-hook and github-hook